src/NServiceBus.Transport.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -63,7 +63,7 @@ public ConnectionFactory(ConnectionConfiguration connectionConfiguration, X509Ce
             }
         }
 
-        public IConnection CreatePublishConnection() => CreateConnection("Publish");
+        public IConnection CreateMessagesConnection() => CreateConnection("Messages");
```

Comment:
I don't really care for this change, because the connection name is visible in the RabbitMQ management UI.

This means we go from seeing that the connection is used for publishing, to "messages" which I think is too generic. All connections to a message broker are for messages!

Reply:
What about the "Administration" connection?